src/js/common.js

```diff
@@ -63,7 +63,23 @@ module.exports.getBinaryExt = (searchableName) => lookup[searchableName].binaryE
 module.exports.getInstallerExt = (searchableName) => lookup[searchableName].installerExtension;
 
 // gets the Supported Version WITH PATH when you pass in 'searchableName'
-module.exports.getSupportedVersion = (searchableName) => lookup[searchableName].supported_version;
+module.exports.getSupportedVersion = (searchableName) => {
+  let supported_version = lookup[searchableName].supported_version;
+  if (typeof supported_version === 'object') {
```

Comment:
`supported_version === 'object'` - This is why I weep at dynamic typing ;-).  V2 of the website - I vote we go Typescript

Reply:
agreed :D